packages/libvips/build.sh

```diff
@@ -18,4 +19,15 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 termux_step_pre_configure() {
 	termux_setup_gir
 	termux_setup_glib_cross_pkg_config_wrapper
+
+	export TERMUX_MESON_ENABLE_SOVERSION=1
+}
+
+termux_step_post_massage() {
+	# Do not forget to bump revision of reverse dependencies and rebuild them
+	# after SOVERSION is changed.
+	local _GUARD_FILE="lib/libvips.so.42"
+	if [ ! -e "${_GUARD_FILE}" ]; then
+		termux_error_exit "Error: file ${_GUARD_FILE} not found."
+	fi
```

Comment:
- I just dealt with something very similar in #24059,
consider using something like the SOGUARD I added in that PR, that way the new SO version gets shown together with the error.
https://github.com/termux/termux-packages/blob/821f2ca442efc9cedbbc0d651ee1440f190184d1/packages/libxml2/build.sh#L33-L38

Reply:
Thanks, that's a bit neater! It's unlikely that libvips will bump its ABI version anytime soon, but it might still be useful for debugging purposes.

Applied with commit fc3b3095342ec9183be65ce713b86aa64d45616d.

Reply:
Ah, looking at the PRs at this repo, it seems that PRs are not squashed merged? Let me squash commit fc3b3095342ec9183be65ce713b86aa64d45616d into 935971eed6c0a335efd571686349440699359290, which should hopefully also fix that lint check.

Reply:
Yeah we usually prefer PR authors to handle the squashing of their commits.
Sometimes multiple commits are warranted for a PR and we prefer letting the author determine what should in what commit, and with what commit message.

Should be a simple `git rebase -i HEAD~2`, `pick`ing the, `squash`ing the second.

Reply:
... fixed with commit 391a57be39be18fd18276d5b0a51b43030f50629. :)